db/models.py

```diff
@@ -50,3 +52,61 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
The `user` field in the `Order` model references a `User` model. Ensure that the `User` model is defined in the same file or imported correctly from another module.